testing/firefox/APKBUILD

```diff
@@ -10,9 +10,13 @@ url="http://www.firefox.com"
 arch="all !aarch64"
 license="GPL LGPL MPL"
 depends=""
-depends_dev="
+makedepends="$depends_dev
 	alsa-lib-dev
+	autoconf2.13
```

Comment:
Those dependencies use to be in "depends_dev" and were included in "makedepends" with "$depends_dev".